src/Components/Header/TopNav/Dropdown.js

```diff
@@ -0,0 +1,33 @@
+import * as React from 'react';
+import Box from '@mui/material/Box';
+import ClickAwayListener from '@mui/base/ClickAwayListener';
+import ArrowDropDownIcon from '@mui/icons-material/ArrowDropDown';
+export default function Dropdown() {
+  const [open, setOpen] = React.useState(false);
+
+  const handleClick = () => {
+    setOpen((prev) => !prev);
+  };
+
+  const handleClickAway = () => {
+    setOpen(false);
+  };
```

Comment:
Since these arrow functions both have a one-line body, you can write each of them in on line without the need for curly brackets.